src/Controller/SubscriptionController.php

```diff
@@ -136,7 +136,9 @@ public function subscribe($entity_type_id, EntityInterface $group, OgMembershipT
       return new RedirectResponse($group->toUrl()->setAbsolute(TRUE)->toString());
     }
 
-    if (!$this->ogAccess->userAccess($group, 'subscribe', $user) && !$this->ogAccess->userAccess($group, 'subscribe without approval', $user)) {
+    $subscribe = $this->ogAccess->userAccess($group, 'subscribe');
+    $subscribeWithoutApproval = $this->ogAccess->userAccess($group, 'subscribe without approval');
```

Comment:
```suggestion
    $subscribe_without_approval = $this->ogAccess->userAccess($group, 'subscribe without approval');
```

Reply:
@amitaibu, changed the variable to snake case in the if as well. fyi, you can now also give code suggestions on multiple lines at once by clicking and dragging over the lines you want.